pages/__tests__/index.test.ts

```diff
@@ -1,13 +1,21 @@
-import index from '@/pages/index.vue'
-import { render } from '@/testUtils'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { setup, $fetch } from '@nuxt/test-utils-edge'
 
-describe('/index', () => {
-  it('should render without crashing', () => {
+import { describe, test } from 'vitest'
```

Comment:
Why do we need the explicit imports here now?

Reply:
Because this is how Vitest w/ nuxt stuff is done according to the getting started. and also explicit imports are better than global implicits :)